plugins/out_stackdriver/stackdriver.c

```diff
@@ -341,6 +341,93 @@ static flb_sds_t get_str_value_from_msgpack_map(msgpack_object_map map,
     return ptr;
 }
 
+/* parse_monitored_resource is to extract the monitoired resource labels
+ * from "logging.googleapis.com/monitored_resource" in log data
+ * and append to 'resource'/'labels' in log entry.
+ * Monitored resource type is already read from resource field in stackdriver
```

Comment:
I would expect `logging.googleapis.com/monitored_resource` also accept a field `type` though - is there any reason you propose not including it?

Note we could defer the decision then as we may expand the field to support under `logging.googleapis.com/monitored_resource`, so it's more of a question on whether you thought about this.

Reply:
Discussed offline. Currently we will not include type field. The type field will be added as a long term solution.